timesketch/frontend-ng/src/views/Sketch.vue

```diff
@@ -208,13 +211,13 @@ limitations under the License.
 
       <!-- Left panel -->
       <v-navigation-drawer
-        v-show="showLeftPanel && hasTimelines && !isArchived"
+        v-model="showLeftPanel"
```

Comment:
This change will allow to have an opened left panel on an archived sketch. Do we want/need to have the left panel active on an archived sketch?

This results in an odd view like this:
![image](https://github.com/google/timesketch/assets/99879757/d02dc439-2f57-4e34-be03-5ee0c334d536)

Reply:
Good catch. Fixed now. Same behaviour and design for the archived page.